harness/determined/_train_context.py

```diff
@@ -134,7 +134,7 @@ class DistributedContext:
     """
     DistributedContext extends all TrialContexts and NativeContexts under
     the ``context.distributed`` namespace. It provides useful methods for
-    effective multi-slot (parallel and distributed) training.
+    effective distributed training.
```

Comment:
We can use this context object also in parallel training. The name `DistributedContext` itself isn't very precise. :)

Reply:
We've been trying to remove the term "parallel training" from the docs / public APIs, and just use "distributed training" for any situation where multiple GPUs are being used.